udata_gouvfr/views.py

```diff
@@ -156,6 +156,16 @@ def nec_mergitur():
                         nb_displayed_datasets=NB_DISPLAYED_DATASETS)
 
 
+@blueprint.route('/openfield16')
+def openfield16():
```

Comment:
Now that we have a couple of custom views doing the same thing, maybe we should refactor a bit?